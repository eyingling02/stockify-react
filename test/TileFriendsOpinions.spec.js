import React from 'react'
import TileFriendsOpinions from '../js/TileFriendsOpinions.jsx'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

test('FriendsOpinions snapshot test', () => {
  const component = shallow(<TileFriendsOpinions />)
  const tree = shallowToJson(component)
  expect(tree).toMatchSnapshot()
})
