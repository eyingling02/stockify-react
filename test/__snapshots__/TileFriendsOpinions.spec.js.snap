exports[`test FriendsOpinions snapshot test 1`] = `
<div
  className="outer">
  <div
    className="buy">
    <a
      href="javascript:;">
      UP
    </a>
  </div>
  <div
    className="inner">
    <a
      href="livingroom.html"
      target="_blank">
      <h2 />
      <p />
      <p />
    </a>
  </div>
  <div
    className="sell">
    <a
      href="javascript:;">
      DOWN
    </a>
  </div>
</div>
`;
